ckanext/spark/templates/home/snippets/search.html

```
@@ -0,0 +1,19 @@
+{% ckan_extends %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)